co2eq/meal.js

```diff
@@ -80,7 +84,7 @@ const CARBON_INTENSITY = { // kgCO2eq / kg
   'Olives': 0.63, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
   'Peppers': 0.66, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
   // fruits
-  'Fruit': 1.1, // http://www.greeneatz.com/foods-carbon-footprint.html
+  'Tomatoes': 1.1, // http://static.ewg.org/reports/2011/meateaters/pdf/methodology_ewg_meat_eaters_guide_to_health_and_climate_2011.pdf page 23
```

Comment:
Let's not replace existing items as this would currently break past meals - feel free to add it if needed

Reply:
The original source (http://www.greeneatz.com/foods-carbon-footprint.html) changed some of the names from the source. "Vegetables" is simply incorrect information, the source had that number for Broccoli and greaneatz arbitrarily changed the name.